tools/netlist/include/CombLoops.h

```diff
@@ -0,0 +1,222 @@
+#ifndef COMBLOOPS_H
+#define COMBLOOPS_H
+
+/*
```

Comment:
IANAL but: I think it's good to leave the original license here but since you translated the source it's now also copyrighted by you and so it's fine to also put the standard license header on this file (and we don't need to put it in external/).